source/io/module_javaScriptInvoke.js

```diff
@@ -139,13 +139,13 @@
             return jsRefNumCookieCounter;
         };
 
-        var dataReadJavaScriptRefNum = function (javaScriptRefNumPointer) {
-            var cookie = Data_ReadJavaScriptRefNum(javaScriptRefNumPointer);
+        Module.javaScriptInvoke.dataReadJavaScriptRefNum = function (javaScriptValueRef) {
```

Comment:
It was my understanding that functions prefixed with data are functions that directly interact with data pointers instead of valueRef's. By the way, we marked all functions in `Module.eggShell` that are prefixed with data as deprecated. Maybe we should remove the data prefix from these two?

Reply:
Agreed that we should remove the data prefix, eggShell should just be an alias to this function.

Reply:
I think we could remove the Data_ReadJavaScriptRefNum and Data_WriteJavaScriptRefNum calls as well. 

What they do is turn a labview type (JSRefNum) into a JS numeric representation and back. We already have a function that turns things into numeric representations and back, the readDouble and writeDouble functions. Those should instead be updated to support this type in addition to the other types they support that can be represented as JS numerics.

I don't think I'll push for that change here, but hopefully overtime we can use this philosophy to shrink the API surface area. It's definitely gotten too verbose.

Reply:
Removed "data" prefix in the dataReadJavaScriptRefNum, dataWriteJavaScriptRefNum APIs.